### DISCUSSION TITLE
How to stop notifications when appliance door opens/closes?

### DISCUSSION THREAD

**Discussion by MathieuN** [thoukydides/homebridge-homeconnect#183] [Q&A]
Each time someone opens or closes the door of the diswasher or the washing machine, it triggers a notification on my phone (home app). Anyway to disable those? Thanks

**Comment by thoukydides** [comment]
This is probably the most frequently asked… and answered… question about this plugin.

Try looking at issue #132 and discussion #165.

**Comment by thoukydides** [comment]
[v0.32.0](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v0.32.0) provides per-appliance configuration options to remove the `Door` services (as well as selectively enabling the `Switch` services for appliance mode settings and the `Stateless Programmable Switch` services for appliance events).
